core/src/main/java/com/linecorp/armeria/common/StringMultimap.java

```diff
@@ -439,16 +440,20 @@ public final boolean contains(IN_NAME name) {
 
     @Override
     public final boolean contains(IN_NAME name, String value) {
+        return contains(name, val -> AsciiString.contentEquals(val, value));
```

Comment:
Hard to know that `val` and `value` mean quickly. 
```suggestion
        return contains(name, actual -> AsciiString.contentEquals(actual, value));
```